pkg/nimble/Makefile

```
@@ -88,3 +88,6 @@ nimble_scanlist:
 
 nimble_scanner:
 	"$(MAKE)" -C $(TDIR)/scanner
+
+nimble_statconn:
+	"$(MAKE)" -C $(TDIR)/statconn
```

```suggestion
	"$(MAKE)" -C $(TDIR)/statconn -f $(RIOTBASE)/Makefile.base MODULE=nimble_statconn
```

And you can drop Makefile.nimble_statconn